packages/components/src/components/model-signature/model-signature.examples.md

```diff
@@ -60,5 +73,28 @@ export default class Code extends Vue {
     const style = this.color === 'inherit' ? ' style="filter: invert(24%) sepia(70%) saturate(5969%) hue-rotate(316deg) brightness(102%) contrast(102%)"' : '';
     return `<p-model-signature color="${this.color}"${style}></p-model-signature>`;
   }
+
+  blendMode = 'overlay';
+  blendModes = ['normal', 'multiply', 'screen', 'overlay', 'darken', 'lighten', 'color-dodge', 'color-burn', 'hard-light', 'soft-light', 'difference', 'exclusion', 'hue', 'saturation', 'color', 'luminosity'];
+  get blendModeMarkup() {
+    return `<div style="isolation: isolate; background: #00aa3680; display: inline-block; padding: 32px;">
```

Comment:
Could have placed it visually in the middle :)

Reply:
maybe next time ;)